CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v1.3.0
+
+This release includes a number of new features and bugfixes. Like the long-awaited propagation of apps' exit status.
+
+#### New features and UX changes
+
+- Check signatures for stage1 images by default ([#2336](https://github.com/coreos/rkt/pull/2336)).
+ This doesn't affect the following cases:
+  - The stage1 image is already in the store
+  - The stage1 image is in the default directory configured at build time
+  - The stage1 image is the default one and it is in the same directory as the rkt binary
+- Allow downloading of insecure public keys with the `pubkey` insecure option ([#2278](https://github.com/coreos/rkt/pull/2278)).
+- Implement Docker volume semantics ([#2315](https://github.com/coreos/rkt/pull/231://github.com/coreos/rkt/pull/2315)). Docker volumes are initialized with the files in the image if they exist, unless a host directory is mounted there. Implement that behavior in rkt when it runs a Docker converted image.
+
+#### API service
+
+- Return the cgroup when getting information about running pods and add a new cgroup filter ([#2331](https://github.com/coreos/rkt/pull/2331)).
+
+#### Bug fixes
+
+- Propagate exit status from apps inside the pod to rkt ([#2308](https://github.com/coreos/rkt/pull/2308)).
```

Comment:
Yeah, we were undecided about it being a feature or a bug fix... I think it's a bit of both